test/src/main/resources/DeploymentSpecifics.properties

```diff
@@ -71,7 +71,8 @@ testdata.dir=${log.dir}/testdata
 fxf.dir=${testdata.dir}/fxf
 
 # Please mark that the backslashes below are double escaped: first for the value of the property, second time for use in regular expression
-log.dir.match=(?i)([cd]:[\\\\/]temp)|(${log.dir})
+
+log.dir.match=(?i)(([cd]:[\\\\/]temp)|(${log.dir}))
```

Comment:
This has already been fixed in PR #2218.

Reply:
Please revert the changes in log.dir.match and testdata.dir.match, merge master and test again.